analysis/analyzer.py

```diff
@@ -79,61 +84,61 @@ def to_proto(self):
     return network_proto
 
 
-def cost(time_ms):
-  """Assigns a cost to a measure of request latency."""
-  # TODO(garretrieger): implement me.
-  return time_ms
-
+class Analyzer:
+  """Holds context for the analysis of a data set against a set of pfe_methods."""
 
-def to_method_result_proto(method_name, totals):
-  """Converts a set of totals for a method into the corresponding proto."""
-  method_result_proto = result_pb2.MethodResultProto()
-  method_result_proto.method_name = method_name
-  request_size_distribution = distribution.Distribution(
-      distribution.LinearBucketer(5))
-  response_size_distribution = distribution.Distribution(
-      distribution.LinearBucketer(5))
+  def __init__(self, cost_function):
+    self.cost_function = cost_function
```

Comment:
This naming makes me expect this is a function not an object. If you need stateful calculators call it something else, if you don't then actually pass functions.